frappe/public/scss/common/css_variables.scss

```diff
@@ -177,19 +177,18 @@
 	--text-2xl: 20px;
 	--text-3xl: 22px;
 
-	--text-on-blue: var(--blue-600);
-	--text-on-light-blue: var(--blue-500);
+	--text-on-blue: var(--blue-700);
+	--text-on-light-blue: var(--blue-700);
```

Comment:
There should be some difference between both variables😅

Reply:
Background colour is quite lighter for light blue, is this required then?

Reply:
Blue-600 has contrast ratio of 4.18 with same background colour (minimum recommendation is 4.5)

Reply:
Then let's make --blue-600 a little darker than #1579D0 and lighter than --blue-700 #1366AE
lets make --blue-600: #1673C5.  it has contrast ratio of 
<img width="593" alt="image" src="https://user-images.githubusercontent.com/30859809/169762636-c9a9297d-dae4-4bc1-9f96-b9ca1304b1c5.png">


<img width="209" alt="image" src="https://user-images.githubusercontent.com/30859809/169762256-452606f2-b0e2-4138-904c-e18c9c8c80b6.png">

Reply:
This looks good 👍